plugin.xml

```diff
@@ -15,7 +15,7 @@
     </js-module>
 
     <dependency id="cordova-sqlcipher-adapter" url="https://github.com/OutSystems/cordova-sqlcipher-adapter.git#0.1.7-OS7" />
-    <dependency id="cordova-plugin-secure-storage" url="https://github.com/OutSystems/cordova-plugin-secure-storage.git#2.6.8-OS12" />
+    <dependency id="cordova-plugin-secure-storage" url="https://github.com/OutSystems/cordova-plugin-secure-storage.git#tentative-fix-encrypted-sp" />
```

Comment:
Does this need to be merged to `master`? If the change to the KeyStore Plugin is fine in a temporary branch, so should this one.

Once a stable version of KeyStore is created, then a stable version for this can be created.

Reply:
Yup. This PR will remain open until we create a new release version on `cordova-plugin-secure-storage`, pointing to it here.